RELEASE_NOTES/2.62.0.md

```diff
@@ -0,0 +1,270 @@
+<!-- THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY. -->
+
+# Fluid Framework v2.62.0
+
+## Contents
+
+- [🚨 Breaking Changes](#-breaking-changes)
+  - [`asTreeViewAlpha` has been deprecated in favor of `asAlpha`. (#25512)](#astreeviewalpha-has-been-deprecated-in-favor-of-asalpha-25512)
```

Comment:
Hmm I feel this one should be under Deprecations and not Breaking Changes (the `__section: breaking` in the changeset shouldn't have been there). Can we do another regeneration of this with that change easily?

Reply:
@noencke